packages/components/input/input-number.ts

```diff
@@ -100,18 +99,18 @@ export const KBQ_NUMBER_INPUT_VALUE_ACCESSOR: any = {
 })
 export class kbqNormalizeWhitespace {
     /** @docs-private */
-    protected elementRef: ElementRef = inject(ElementRef);
-    /** @docs-private */
-    protected clipboard = inject(Clipboard);
+    protected elementRef: ElementRef<HTMLElement> = inject(ElementRef);
```

Comment:
эта директива применяется только к инпуту?
```suggestion
    protected elementRef: ElementRef<HTMLInputElement> = inject(ElementRef);
```

Reply:
@artembelik в теории может применяться в разных элементах, где нужно заменить тонкий пробел на обычный, поэтому сделал более гибко.

Reply:
тогда будет логичным переместить эту директиву в core (не в рамках этого пр)

Reply:
так как релиза еще не было, не должно возникнуть конфликтов. Перенес в `core`